doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,92 @@
+# Advocacy Ambassador program
+
+Node.js aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that it would like ambassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up-to-date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on, when they would be happy for ambassadors to help share and get
+  early feedback.
```

Comment:
I can't tell if this is saying "ambassadors share topics and also share information about when they would want feedback" (in which case, there needs to be a conjunction) of if it means "when maintainers want feedback on a topic, they will share the topic with ambassadors" (in which case, the comma needs to be removed). I *think* this is supposed to mean the latter so I'll add that as a suggestion:

```suggestion
* Providing a space for collaborators to share new topics/features they
  are working on when they would be happy for ambassadors to help share and get
  early feedback.
```